benchmarks/nel/scripts/datasets/dataset.py

```diff
@@ -393,11 +399,50 @@ def evaluate(
             eval_results.append(spacyfishing_results)
 
         logger.info(dict(cand_gen_label_counts))
-        evaluation.EvaluationResults.report(tuple(eval_results))
+        evaluation.EvaluationResults.report(tuple(eval_results), run_name=run_name, dataset_name=self.name)
 
         self._nlp_best.config["incl_context"] = False
         self._nlp_best.config["incl_prior"] = False
 
+    def compare_evaluations(self, highlight_criterion: str) -> None:
+        """Generate and display table for comparison of all available runs for this dataset.
+        highlight_criterion (str): Criterion to highlight in table. One of ("F-score", "Recall", "Precision").
+        """
+        assert highlight_criterion in ("F-score", "Recall", "Precision"), \
+            "Criterion must be one of ('F-score', 'Recall', 'Precision')"
+
+        header: Optional[List[str, ...]] = None
+        rows: List[List[str, ...]] = []
+        dir_path = Path(os.path.abspath(__file__)).parent.parent.parent / "evaluation" / self.name
+
+        for path in dir_path.glob("*.csv"):
+            with open(path, "r") as csv_file:
+                csv_reader = csv.reader(csv_file)
+                _header = next(csv_reader)
+                if header is None:
+                    header = ["Run", *_header]
+                rows.extend([[path.stem, *row] for row in csv_reader])
+
+        table = prettytable.PrettyTable(field_names=header)
+        rows = sorted(rows, key=operator.itemgetter(0, 1))
+        highlight_crit_idx = header.index(highlight_criterion)
+        max_crit_non_cand_gen = .0
+        max_crit_cand_gen = .0
+        for row in rows:
+            if row[1] != "Candidate Gen.":
+                max_crit_non_cand_gen = max(max_crit_non_cand_gen, float(row[highlight_crit_idx]))
+            else:
+                max_crit_cand_gen = max(max_crit_cand_gen, float(row[highlight_crit_idx]))
+        for row in rows:
+            if (
+                row[1] != "Candidate Gen." and float(row[highlight_crit_idx]) >= max_crit_non_cand_gen or
+                row[1] == "Candidate Gen." and float(row[highlight_crit_idx]) >= max_crit_cand_gen
+            ):
+                for i in range(len(row)):
+                    row[i] = '\033[1m' + row[i] + '\033[0m'
```

Comment:
You can change it to reverse video like this, though since it seems to only affect the contents and the not entire row it doesn't look great. Underline (`4`) looks a bit better. Honestly it's also fine as-is though. 

```suggestion
                    row[i] = '\033[7m' + row[i] + '\033[0m'
```

Reply:
Going with underline now, I agree it looks better. Thanks for the recommendation!